static/styles/night_mode.scss

```diff
@@ -698,3 +698,9 @@ on a dark background, and don't change the dark labels dark either. */
         background-color: hsla(0, 0%, 0%, 0.2);
     }
 }
+
+@media (prefers-color-scheme: dark) {
+    .dark-preference-checker {
```

Comment:
We could up with a better name for this.  Made `color-scheme-automatic` would be clearer?